src/Autocomplete/tests/Fixtures/Autocompleter/CustomAttributesProductAutocompleter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\UX\Autocomplete\Tests\Fixtures\Autocompleter;
+
+class CustomAttributesProductAutocompleter extends CustomProductAutocompleter
+{
+    public function getAttributes(object $entity): array
+    {
+        return ['disabled' => true];
```

Comment:
```suggestion
        return [
            'disabled' => true,
            'text' => 'This value should be replaced with the result of getText()',
            'value' => 'This value should be replaced with the result of getValue()',            
        ];
```

That way we also prevent any future regression on method precedence

Reply:
Crap, I thought of it last night and wanted to implement it this morning, but you've been too fast ^^